### PR TITLE
adds community tasks

### PR DESCRIPTION
**Pull request by pradeepitm12**
this adds community tasks to operator as listed below
 - Helm install chart from source
 - Helm install chart from repo--
 - Skopeo--
 - Trigger Jenkins Job--
 - Git-cli
 - https://github.com/tektoncd/catalog/tree/master/task/pull-request
 - https://github.com/tektoncd/catalog/tree/master/task/kubeconfig-creator

Note: productized image for skopeo

Test:
set

export IMAGE_ADDONS_SKOPEO_COPY=registry.redhat.io/rhel8/skopeo

and run

operator-sdk up local --namespace=""

### CODE REVIEW

**Comment by pradeepitm12** [comment]
cc: @savitaashture  @vdemeester @nikhil-thomas

File: deploy/operator.yaml [hunk 1]
```diff
@@ -26,6 +26,8 @@ spec:
           value: ""
         - name: IMAGE_ADDONS_PARAM_KN_IMAGE
           value: registry.redhat.io/openshift-serverless-1/client-kn-rhel8:0.13.2
+        - name: IMAGE_ADDONS_SKOPEO_COPY
+          value: registry.redhat.io/rhel8/skopeo
```
Comment (vdemeester) [inline deploy/operator.yaml:30]: naive question : how is the *magically* used ? :stuck_out_tongue_closed_eyes:

Comment (pradeepitm12) [reply deploy/operator.yaml]: :stuck_out_tongue_closed_eyes: it took some time for me to get it, 
but the name of the env variable is the main magic and else is taken care [here](https://github.com/openshift/tektoncd-pipeline-operator/blob/2f271f6761de79fb494c4c8a570486f6e1c28217/pkg/utils/transform/transform.go#L277) and then [here](https://github.com/openshift/tektoncd-pipeline-operator/blob/2f271f6761de79fb494c4c8a570486f6e1c28217/pkg/utils/transform/transform.go#L312)

**Review by vdemeester — APPROVED** [review]
/meow

**Review by khrm — APPROVED** [review]
/lgtm

**Pull request merged**